apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsFilters/AlertsFilters.tsx

```diff
@@ -77,7 +78,7 @@ export const AlertsFilters: FunctionComponent<{
       <div className={`flex gap-x-2`}>
         {filters.map(({ title, accessor, options }) => (
           <MultiSelect
-            key={title}
+            key={keyFactory(title, filter?.[accessor])}
```

Comment:
Out of curiosity, how does that ensure the re-render?

Reply:
`filter` is from the URL. Since its part of the key whenever that value changes it triggers a re-render. Can also be used outside of iteration for things like re-triggering on-mount animations.